### PR TITLE
Update gradle.properties to prevent OOM

### PR DESCRIPTION
**Pull request by rexhoffman**
works around issues like: https://github.com/robolectric/robolectric/actions/runs/9471569024/job/26095077660

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -12,5 +12,8 @@ kotlin.stdlib.default.dependency=false
 # https://docs.gradle.org/current/userguide/performance.html#enable_the_build_cache
 org.gradle.caching=true
 
+#Up gradle's max memory due to ooms during compilation.
```
Comment (hoisie) [inline gradle.properties:15]: nit: space between `#` and `Up`

Comment (MGaetan89) [reply gradle.properties:15]: What about:
```suggestion
# Increase Gradle's max memory due to OutOfMemoryException during compilation
```

**Review by hoisie — APPROVED** [review]

**Comment by hoisie** [comment]
Whoops I did not use squash and merge. Will be more careful next time.

**Comment by MGaetan89** [comment]
If only squash & merge should be used, why not disable the other options in the repository settings?
Screenshot 2024-06-13 at 17 18 10

**Pull request merged**